subprojects/gradle/runner/report/src/main/kotlin/com/avito/android/runner/report/internal/ReportImpl.kt

```diff
@@ -0,0 +1,40 @@
+package com.avito.android.runner.report.internal
+
+import com.avito.android.runner.report.Report
+import com.avito.report.model.AndroidTest
+import com.avito.report.model.TestStaticData
+
+internal class ReportImpl(
+    private val inMemoryReport: InMemoryReport,
+    private val avitoReport: AvitoReport?, // todo should be generic reports from config
```

Comment:
It's bad that api makes us able to set `inMemoryReport=null` and `useInMemoryReport=false` and it will lead to crash